include/Types.hpp

```diff
@@ -27,4 +28,5 @@ namespace blitzdg {
     using index_tensor3_type = tensor3_type<index_type>;
     using real_tensor3_type = tensor3_type<real_type>;
 
+    using index_hashmap = std::unordered_map<blitzdg::index_type, blitzdg::index_vector_type>;
 }
```

Comment:
You don't need to include the blitzdg:: namespaces as you are already in that namespace.